hmp/preprocessing.py

```diff
@@ -358,6 +360,7 @@ def __init__(  # noqa  # Might need to be refactored.
 
         data.attrs["sfreq"] = sfreq
         data.attrs["offset"] = offset
+        data.attrs["offset_before"] = offset_before
```

Comment:
Clearer I would say, also for people who might use other things than stimulus and response as interval
```suggestion
        data.attrs["offset_start"] = offset_start
```